common3.json

```diff
@@ -6131,743 +6131,743 @@
     {
       "id" : 61552,
-      "text" : "Sp.atk drop: 味方が「Helping Hand」を選択した"
+      "text" : "Sp.atk drop: Ally chose Helping Hand"
     },
     {
       "id" : 61553,
-      "text" : "Sp.def drop: 対象が?%の確立で特防がダウンする可能性がある"
+      "text" : "Sp.def drop: Target has ?% Probability of Sp Def decreasing"
     },
     {
       "id" : 61554,
-      "text" : "Sp.def drop: 味方が「Helping Hand」を選択した"
+      "text" : "Sp.def drop: Ally chose Helping Hand"
     },
     {
       "id" : 61555,
-      "text" : "Accuracy drop: 対象が?%の確立で命中がダウンする可能性がある"
+      "text" : "Accuracy drop: Target has ?% Probability of Accuracy decreasing"
     },
     {
       "id" : 61556,
-      "text" : "Accuracy drop: 味方が「Helping Hand」を選択した"
+      "text" : "Accuracy drop: Ally chose Helping Hand"
     },
     {
       "id" : 61557,
-      "text" : "Confusion: 対象が?%の確立で混乱する可能性がある"
+      "text" : "Confusion: Target has ?% Probability of being confused"
     },
     {
       "id" : 61558,
-      "text" : "Confusion: 味方が「Helping Hand」を選択した"
+      "text" : "Confusion: Ally chose Helping Hand"
     },
     {
       "id" : 61559,
-      "text" : "Sacred Fire?: 対象が?%の確立で火傷する可能性がある"
+      "text" : "Sacred Fire?: Target has ?% Probability of being burned"
     },
     {
       "id" : 61560,
-      "text" : "Sacred Fire?: 味方が「Helping Hand」を選択した"
+      "text" : "Sacred Fire?: Ally chose Helping Hand"
     },
     {
       "id" : 61561,
       "text" : "Sacred Fire?: User is frozen"
     },
     {
       "id" : 61562,
-      "text" : "Fire Spin: 対象が交代可能状態である"
+      "text" : "Fire Spin: Target has an changeable status"
     },
     {
       "id" : 61563,
-      "text" : "Fire Spin: 対象が毒状態である"
+      "text" : "Fire Spin: Target is poisoned"
     },
     {
       "id" : 61564,
-      "text" : "Fire Spin: 対象がBadly Poisoned状態である"
+      "text" : "Fire Spin: Target is Badly Poisoned"
     },
     {
       "id" : 61565,
-      "text" : "Fire Spin: 対象が火傷状態である"
+      "text" : "Fire Spin: target is burned"
     },
     {
       "id" : 61566,
-      "text" : "Fire Spin: 対象が宿られ状態である"
+      "text" : "Fire Spin: target is seeded"
     },
     {
       "id" : 61567,
-      "text" : "Fire Spin: 対象が混乱状態である"
+      "text" : "Fire Spin: target is confused"
     },
     {
       "id" : 61568,
-      "text" : "Fire Spin: 対象がAttract状態である"
+      "text" : "Fire Spin: target is infatuated"
     },
     {
       "id" : 61569,
-      "text" : "Fire Spin: 対象が呪い状態である"
+      "text" : "Fire Spin: target is cursed"
     },
     {
       "id" : 61570,
-      "text" : "Fire Spin: 対象が滅び状態である"
+      "text" : "Fire Spin: target is under perish song"
     },
     {
       "id" : 61571,
-      "text" : "Fire Spin: 対象が凍り状態である"
+      "text" : "Fire Spin: Target is frozen"
     },
     {
       "id" : 61572,
-      "text" : "Fire Spin: 対象が交代不可能状態である"
+      "text" : "Fire Spin: Target has an unchangeable status"
     },
     {
       "id" : 61573,
-      "text" : "自己解凍炎系　自分が凍り状態である"
+      "text" : "自己解凍炎系　user is frozen"
```

Comment:
this is just a general comment for @StarsMmd but fal said they were uncertain about it and that it meant "self extracting flame", but from the context of the near ones, this just sounds like it means "Burning", wouldn't you agree? cause it seems the AI is weighting some status effects